uefi/src/prelude.rs

```diff
@@ -6,5 +6,6 @@ pub use crate::{cstr16, cstr8, entry, Handle, ResultExt, Status, StatusExt};
 
 // Import the basic table types.
 pub use crate::table::boot::BootServices;
+#[allow(deprecated)]
```

Comment:
Deprecated public export statements are not supported yet. https://github.com/rust-lang/rust/issues/30827